functions/src/platform/assistant/handler.js

```diff
@@ -24,17 +24,22 @@ const userUIDMiddleware = require('./middlewares/user-uid');
 module.exports = (actionsMap) => {
   const app = dialogflow();
 
-  const dashbot = dashbotBuilder(functions.config().dashbot.key, {
-    // it could be more useful if we would get callback on error and pass log through our logger
-    // but currently it uses console.log to log error
-    //
-    // more details in source:
-    // https://github.com/actionably/dashbot/blob/33376ff81af3962eb58ad8ebabc91827134333c5/src/make-request.js#L22
-    //
-    printErrors: false,
-  }).google;
-
-  dashbot.configHandler(app);
+  if (_.has(functions.config(), ['dashbot', 'key'])) {
+    const dashbot = dashbotBuilder(functions.config().dashbot.key, {
+      // it could be more useful if we would get callback on error and pass log through our logger
+      // but currently it uses console.log to log error
+      //
+      // more details in source:
+      // https://github.com/actionably/dashbot/blob/33376ff81af3962eb58ad8ebabc91827134333c5/src/make-request.js#L22
+      //
+      printErrors: false,
+    }).google;
+
+    dashbot.configHandler(app);
+  } else {
+    console.log('\n');
+    warning('\nWARNING : dashbot key missing\n');
```

Comment:
you don't need `\nWARNING :` when you call `warning()`, it will add that keyword to logs.